src/mesh/mesh.cpp

```diff
@@ -77,6 +77,8 @@ Mesh::Mesh(ParameterInput *pin, ApplicationInput *app_in, Packages_t &packages,
       default_pack_size_(pin->GetOrAddInteger("parthenon/mesh", "pack_size", -1)),
       // private members:
       num_mesh_threads_(pin->GetOrAddInteger("parthenon/mesh", "num_threads", 1)),
+      comm_vector_length_(
+          pin->GetOrAddInteger("parthenon/mesh", "comm_vector_length", 1)),
```

Comment:
do you have a sense for what the right default size is here?

Reply:
Not at all, in fact I am not sure if it makes any difference. That being said, the Kokkos default is one, so I just put that here.